src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx

```diff
@@ -14,6 +14,9 @@ import {
     ATTRIBUTE_DISPLAY_FORM_IDENTIFIER,
 } from "./utils";
 
+const delayOffset = 250; // Magic constant inside Goodstrap FLEX_DIMENSIONS_THROTTLE :(
```

Comment:
Is it possible and worthy to import the constant?

Reply:
there is no constant in GS and in this test it can change if tests will start to fail